tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoGetConf.java

```diff
@@ -119,6 +119,9 @@ public void runCommand(String[] args, boolean tsqlMode) throws Exception {
     } else if (hostName == null && port == null) {
       tajoClient = new TajoClientImpl(ServiceTrackerFactory.get(tajoConf));
     }
+    if (!tajoClient.isConnected()) {
```

Comment:
Actually, this class does not need TajoClient. I think that removing tajoClient is a better solution. So, you don't need the below ``System.out.print`` statement.